nucypher/policy/policies.py

```diff
@@ -8,15 +8,16 @@
 import maya
 from bytestring_splitter import BytestringSplitter, VariableLengthBytestring
 from constant_sorrow.constants import NOT_SIGNED, UNKNOWN_KFRAG, FEDERATED_POLICY, UNKNOWN_ARRANGEMENTS
+from nacl.hashlib import blake2b
```

Comment:
Hmm, what's this for?

Reply:
Artifact from playing with deterministic arrangement IDs, I will remove it.